src/PKSim.Infrastructure/Services/ImportObservedDataTask.cs

```diff
@@ -227,7 +227,7 @@ private void addObservedData(Func<IReadOnlyList<ColumnInfo>> importConfiguration
 
          var importedObservedData = _dataImporter.ImportDataSets((IReadOnlyList<MetaDataCategory>)metaDataCategories, importConfiguration(), dataImporterSettings);
 
-         if (importedObservedData.DataRepositories == null || importedObservedData.Configuration == null) return;
+         if (importedObservedData.DataRepositories == null || importedObservedData.Configuration == null || string.IsNullOrEmpty(importedObservedData.Configuration.Id)) return;
```

Comment:
May I ask how we can have  a configuration that has no id?

Reply:
Yep, there was a wrong initialization of the configuration on the Core side. I fixed it now (since it was quite obvious once found) and I am planning to write a test for this later today after I finish some other more pressing issues. I would say we should actually close this PR even without merging this change. I am not sure we should ignore a bad configuration here since it is better to crash if unintended use rather than silently ignore.

Reply:
agreed. closing